onnx/defs/function.h

```diff
@@ -57,6 +57,18 @@ class FunctionBuilderRegistry {
   static Common::Status function_builder_##counter##_status = \
       FunctionBuilderRegistry::OnnxInstance().Register(function_builder);
 
+// Helper function for register nodes in
+// a FunctionProto. Attributes need to be
+// registered separately.
+Common::Status BuildNode(
```

Comment:
This is similar to a function in helper.py. If C++ doesn't have such a helper module, we should create one. Otherwise, it'd be better to reuse it.

Reply:
Double checked and there is no node constructing functions in C++

Reply:
In this way, can we move this function to a new common file please? say, ModelHelper.h/cc in common folder.